### ISSUE TITLE
[Adaptive Card iOS - TooltipTestCard.json]: Clear text button under 'Inline action with' is not accessible through the swipe gesture.

### ISSUE THREAD

**Issue by jwoo-msft** [microsoft/AdaptiveCards#7026]
https://microsoft.visualstudio.com/OS/_workitems/edit/37857113

**Comment by vagpt** [comment]
Hi @jwoo-msft thanks for the updates.

We are not able to test this issue right now as elements under cards are not accessible via swipe gesture hence, reactivating this issue.

**Build Version:** Build Version -1.0(2.4.0-beta.20220718.1)

**Attachment:**

https://user-images.githubusercontent.com/95913748/180012475-84ca90cb-4756-47d3-a330-e3344bf848ff.mp4

**Comment by jwoo-msft** [comment]
I've updated sample, so you can test it.

Please use samples/v1.3/Scenarios/InputsWithValidation.json.

There is an issue with test app, and it wouldn't scroll as vo navigates down; this is a test app issue; I'm working on fixing it to improve the experience.

meanwhile, to test it, scroll to input.text with inline action, then enable vo, and test it by tap on to the input.text.

**Comment by vagpt** [comment]
This issue is not repro on the release 179, hence adding A11yTTValidated tag.
